docs/source/user-guide/configuration/custom-env-and-pkg-locations.rst

```diff
@@ -0,0 +1,31 @@
+========================================================
+Using Custom Locations for Environment and Package Cache
+========================================================
+
+For any given conda installation, the two largest folders in terms of
+disk space are often the ``envs`` and ``pkgs`` folders
+that store created environments and downloaded packages, respectively.
+If the location where conda is installed has limited disk
+space and another location with more disk space is available on the
+same computer, we can change where conda saves its environments and
+packages with the settings ``envs_dirs`` and ``pkgs_dirs``, respectively.
+
+Assuming conda is installed in the user's home directory and the
+the folder ``/volume/user`` with more disk space is writable,
```

Comment:
I'm wondering, is there is a good canonical example for a mounted directory that is less generic looking than `/volume/user`?

Reply:
Perhaps one of these would be better?

- `/media/nfs/home/user`
- `/nfs/home/user`
- `/volume/home/user`

They all have added `home/user` to express that this is a mounted home directory.

Reply:
Let's use `/nfs/home/user` to it's explicit

Reply:
```suggestion
the folder ``/nfs/volume/user`` with more disk space is writable,
```